docs/layouts/shortcodes/generated/forst_configurable_configuration.html

```diff
@@ -0,0 +1,168 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>state.backend.forst.block.blocksize</h5></td>
+            <td style="word-wrap: break-word;">4 kb</td>
+            <td>MemorySize</td>
+            <td>The approximate size (in bytes) of user data packed per block. The default blocksize is '4KB'.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.block.cache-size</h5></td>
+            <td style="word-wrap: break-word;">8 mb</td>
+            <td>MemorySize</td>
+            <td>The amount of the cache for data blocks in ForSt. The default block-cache size is '8MB'.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.block.metadata-blocksize</h5></td>
+            <td style="word-wrap: break-word;">4 kb</td>
+            <td>MemorySize</td>
+            <td>Approximate size of partitioned metadata packed per block. Currently applied to indexes block when partitioned index/filters option is enabled. The default blocksize is '4KB'.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.bloom-filter.bits-per-key</h5></td>
+            <td style="word-wrap: break-word;">10.0</td>
+            <td>Double</td>
+            <td>Bits per key that bloom filter will use, this only take effect when bloom filter is used. The default value is 10.0.</td>
```

Comment:
nit:
 bloom filter -> the bloom filter
take - > takes
when -> when the

Why is this decimal- what does the fraction mean?

Reply:
> Why is this decimal- what does the fraction mean?

Because multiple keys will generate one bloom filter, the bits in that filter will be this decimal multiplies the number of keys, round up to integer.
 ---- Need to read the concept of bloom filter first, then you can understand this option well.